go/vt/vtgate/planbuilder/operators/queryprojection.go

```diff
@@ -97,7 +99,7 @@ func (aggr Aggr) NeedsWeightString(ctx *plancontext.PlanningContext) bool {
 
 func (aggr Aggr) GetTypeCollation(ctx *plancontext.PlanningContext) evalengine.Type {
 	if aggr.Func == nil {
-		return evalengine.Type{}
+		return evalengine.NewType(sqltypes.Unknown, collations.Unknown)
```

Comment:
@systay What do you think of a new evalengine.NewUnknownType() function to replace this one and all other cases like this with?